src/common/flowConfig.ml

```diff
@@ -61,6 +62,7 @@ let default_module_system = Node
 let default_options root = {
   moduleSystem = default_module_system;
   module_name_mappers = [];
+  module_file_exts = [];
```

Comment:
Ahh, this is a good point. We want users to start from an empty list if they specify *something*, but get the default if they specify *nothing*.

The more I think about this, the more it makes me believe we should move the "default" value into something like a `set_default` function in the `OptionsParser.option_entry` record so that it can be called when any option is *not* encountered.

Anyway, don't worry too much about doing this now -- it's more cleanup work than pertinent to this PR, so I don't think it's worth shaving that yak here and now.